packages/@lwc/engine-core/src/3rdparty/snabbdom/snabbdom.ts

```diff
@@ -73,7 +74,12 @@ function removeVnodes(parentElm: Node, vnodes: VNodes, startIdx: number, endIdx:
     }
 }
 
-export function updateDynamicChildren(parentElm: Node, oldCh: VNodes, newCh: VNodes) {
+export function updateDynamicChildren(
+    parentElm: Node,
+    oldCh: VNodes,
+    newCh: VNodes,
+    before: Node | null = null
```

Comment:
I will be very skeptic about changes in this file... they must be very well justified. Up to this point, I haven't see anything in light dom that justify changes here.